.github/CODEOWNERS

```diff
@@ -13,13 +13,13 @@
 # Assigning everyone who owns a package to the common/changes review
 /common/changes @wilmaier @calebmshafer @JoeZman @archana-maharjan @aruniverse @diegopinate @JValiunas @michaalx @grigasp @michaelbelousov
```

Comment:
While we're switching things around, let's fix this. I don't like everyone being added just because there's a changelog.  Can you update it to explicitly set each package? (Gross but the only way I know to do it)

Reply:
Just to confirm, you wanted to revert changes made in #66

Reply:
Hmm idk if this will work. With the major bump for markup-frontstage, it doesnt show up in this [dir](https://github.com/imodeljs/viewer-components-react/tree/master/common/changes/%40bentley) causing the validator to fail

Reply:
Yea, I've been running into this when trying to make a change in iModel.js repo.  Not entirely sure how i want to handle it.

Technically it should be a valid codeowners file and it's only failing the validator that I have in place.  We can just remove that check from the validator.

Reply:
So i removed the `files` check. Added the `owners` check. but thats failing and im guessing the token has expired/needs to get updated. 
![image](https://user-images.githubusercontent.com/11051042/118156905-e4a88a00-b3e7-11eb-8fe8-56479d9cce68.png)

Reply:
We should update it to match the iModel.js one to be, 

```
with:
  checks: "duppatterns,syntax"
  experimental_checks: "notowned"
  # GitHub access token is required only if the `owners` check is enabled
  github_access_token: "${{ secrets.OWNERS_VALIDATOR_GITHUB_SECRET }}"
```

It does not currently include the OWNERS one but that's something I need to fix.

Reply:
I'll add whatever token we need to have.